### PR TITLE
fix(labs): updates successfully completed message

### LINKED ISSUE #2295: Successfully completed lab shows translation key instead of actual text
## 🐛 Bug Report
When completing a lab, it shows the translation key instead of the actual text. 

## To Reproduce
1. create a lab
2. complete the lab
3. see the key says lab.successfullyCompleted

## Expected behavior
The message should say `Successfully completed ${type} for ${patient}`

### PR DESCRIPTION
**Pull request by gaweki**
Fixes #2295 

Fixes button text view lab.

**Changes proposed in this pull request:**

- Add successfullyCompleted to enUs translations

**Newly added dependencies with [Bundlephobia](https://bundlephobia.com/) links:**

NONE

### CODE REVIEW

**Comment by gaweki** [comment]
hello @jackcmeyer,  please review

File: src/shared/locales/enUs/translations/labs/index.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export default {
     filterTitle: 'Filter by status',
     search: 'Search labs',
     successfullyUpdated: 'Successfully updated',
+    successfullyCompleted: 'Succesfully Completed',
```
Comment (blestab) [inline src/shared/locales/enUs/translations/labs/index.ts:7]: May you correct the spelling for _succes**s**fully_ and use lowercase _c_ in the word _completed_ to keep the message consistent with other messages i.e. your message must be changed to "Successfully completed"

**Review by blestab — CHANGES_REQUESTED** [review]
May you correct the spelling for successfully and use lowercase c in the word completed to keep the message consistent with other messages i.e. your message must be changed to "Successfully completed"

File: src/shared/locales/enUs/translations/labs/index.ts (hunk 1 above)
Comment (gaweki) [reply src/shared/locales/enUs/translations/labs/index.ts:7]: okay @blestab

Comment (gaweki) [reply src/shared/locales/enUs/translations/labs/index.ts]: done @blestab

**Review by jackcmeyer — DISMISSED** [review]
The issue describes the expected message to be: 

The message should say `Successfully completed ${type} for ${patient}`

Currently the message says `Successfully completed ${type} ${patient}`

**Comment by gaweki** [comment]
> The issue describes the expected message to be:
> 
> The message should say `Successfully completed ${type} for ${patient}`
> 
> Currently the message says `Successfully completed ${type} ${patient}`

okey i will update it

**Review by blestab — APPROVED** [review]

**Review by fox1t — APPROVED** [review]

**Pull request merged**